docs/README.rst

```diff
@@ -0,0 +1,33 @@
+=========================
```

Comment:
Awesome! 
Really well-toughed file, a guide like this with our decisions is more than needed!